src/Form/FormMapper.php

```diff
@@ -114,9 +114,13 @@ public function add($name, $type = null, array $options = [], array $fieldDescri
 
         if ($name instanceof FormBuilderInterface) {
             $this->formBuilder->add($name);
+            $type = null;
```

Comment:
why is this needed here?

Reply:
Because if the previous condition is met, we need to pass default values to arguments 2 and 3 for `FormBuilder::add()`.